app/locales/en/translation.json

```
@@ -541,6 +541,6 @@
     "baseInputs": "base inputs",
     "temporalInputs": "temporal inputs",
     "environmentalInputs": "environmental inputs",
-    "selectValuesToGenerateScore": "Select values for all metrics to generate a score"
+    "selectValuesToGenerateScore": "Select values for all metrics (individually or via the vector)  to generate a score"
```

My bad - double space...
```suggestion
    "selectValuesToGenerateScore": "Select values for all metrics (individually or via the vector) to generate a score"